### PR TITLE
refactored App with new radiobuttons and checkbox components

### PR DESCRIPTION
**Pull request by Oleh-Li**
please check

### CODE REVIEW

File: src/components/ciSelectPlan/TwoRadioButtonSelectQuestion.js [hunk 1]
```diff
@@ -3,14 +3,16 @@ import { PseudoElement } from "../stdPlanOptions/PseudoElement";
...
 
 export const TwoRadioButtonSelectQuestion = () => {
   return (
     <View style={[styles.sectorWrapper, shadowStyles.boxShadow]}>
       <Text style={styles.questionTitle}>Are you actively at work?</Text>
       <PseudoElement />
       <View style={styles.radioButtonsWrapper}>
-        <View style={styles.radioButtonAndTextWrapper}>
+        <RadioButtonsCiSelectPlan/>
+        {/* <View style={styles.radioButtonAndTextWrapper}>
```
Comment (osavchukdoit) [inline src/components/ciSelectPlan/TwoRadioButtonSelectQuestion.js:15]: Do we need these commented lines?

File: src/components/ciSelectPlan/TwoRadioButtonSelectQuestion.js [hunk 2]
```diff
@@ -33,7 +35,7 @@ const styles = StyleSheet.create({
     borderRadius: 20,
     paddingHorizontal: 15,
     paddingTop: 19,
-    paddingBottom: 22,
+    // paddingBottom: 22,
```
Comment (osavchukdoit) [inline src/components/ciSelectPlan/TwoRadioButtonSelectQuestion.js:38]: And this one?

File: src/components/utils/RadioButtonsCiSelectPlan.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
+import React, { useState } from "react";
+import { StyleSheet, View, TouchableOpacity, Text } from "react-native";
+import { THEME } from "../../styles/theme";
+import { FONTS } from "../../styles/fonts";
+
+export const RadioButtonsCiSelectPlan = () => {
+  const [isValue, setIsValue] = useState(null);
```
Comment (osavchukdoit) [inline src/components/utils/RadioButtonsCiSelectPlan.js:7]: It is better to rename "setIsValue" to "setValue"

File: src/components/utils/RadioButtonsYesNo.js [hunk 4]
```diff
@@ -0,0 +1,80 @@
+import React, { useState } from "react";
+import { StyleSheet, View, TouchableOpacity, Text } from "react-native";
+import { THEME } from "../../styles/theme";
+import { FONTS } from "../../styles/fonts";
+
+export const RadioButtonsYesNo = () => {
+  const [isValue, setIsValue] = useState(null);
```
Comment (osavchukdoit) [inline src/components/utils/RadioButtonsYesNo.js:7]: Same here: checked and setChecked.

File: src/components/utils/RadioButtonsYesNo.js [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+            ]}
+          ></View>
+        </View>
+        <Text style={styles.radioButtonLabel}>No</Text>
+      </TouchableOpacity>
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
```
Comment (osavchukdoit) [inline src/components/utils/RadioButtonsYesNo.js:40]: We can extract this styles object to some external file to use it for RadioButtonsCiSelectPlan.
If there are some different styles - we can override them with some local styles.

**Comment by Oleh-Li** [comment]
fixed all points, please check again

**Review by osavchukdoit — APPROVED** [review]

**Comment by osavchukdoit** [comment]
ok

**Pull request merged**